README.md

```diff
@@ -1,129 +1,112 @@
+concentrate their liquidity around fewer pairs, creating order book thickness
+and competitive FX rates, especially for exotic currency pairs. Unlike USD,
+trading through XRP does not require bank accounts, service fees, counterparty
+risk, or additional operational costs. By using XRP, liquidity providers can
+specialize in certain currency corridors, reduce operational costs, and
+ultimately, offer more competitive FX pricing.
+
+![Flow - Bridged over XRP](images/flow2.png)
+
+# rippled - Ripple server
+`rippled` is the reference server implementation of the Ripple
+protocol. To learn more about how to build and run a `rippled`
+server, visit https://ripple.com/build/rippled-setup/
+
+## [![Build Status](https://travis-ci.org/ripple/rippled.png?branch=develop)](https://travis-ci.org/ripple/rippled)
+
+### License
+`rippled` is open source and permissively licensed under the
+ISC license. See the LICENSE file for more details.
+
+#### Repository Contents
+
+##### ./bin
 Scripts and data files for Ripple integrators.
 
-#### ./build
+##### ./build
 Intermediate and final build outputs.
 
-#### ./Builds
+##### ./Builds
 Platform or IDE-specific project files.
 
-#### ./doc
+##### ./doc
 Documentation and example configuration files.
 
-#### ./src
+##### ./src
 Source code directory. Some of the directories contained here are
 external repositories inlined via git-subtree, see the corresponding
 README for more details.
 
-#### ./test
+##### ./test
 Javascript / Mocha tests.
 
-## License
-Ripple is open source and permissively licensed under the ISC license. See the
-LICENSE file for more details.
+##For more information:
+
+* [Ripple Knowledge Center](https://ripple.com/learn/)
+* [Ripple Developer Center](https://ripple.com/build/)
+* [Ripple Whitepapers & Reports](https://ripple.com/whitepapers-reports/)
+  * [Ripple Consensus Whitepaper](https://ripple.com/consensus-whitepaper/)
+  * [Ripple Solutions Guide](https://ripple.com/files/ripple_solutions_guide.pdf)
+  * [Ripple Executive Summary for Financial Institutions](https://ripple.com/solutions/executive-summary-for-financial-institutions/)
 
-###For more information:
-* Ripple Wiki - https://ripple.com/wiki/
-* Ripple Primer - https://ripple.com/ripple_primer.pdf
-* Ripple Primer (Market Making) - https://ripple.com/ripple-mm.pdf
-* Ripple Gateway Primer - https://ripple.com/ripple-gateways.pdf
-* Consensus - https://wiki.ripple.com/Consensus
+To learn about how Ripple is transforming global payments visit
+[https://ripple.com/contact/](https://ripple.com/contact/)
 
 - - -
 
 Copyright © 2015, Ripple Labs. All rights reserved.
 
-Portions of this document, including but not limited to the Ripple logo, images
-and image templates are the property of Ripple Labs and cannot be copied or
-used without permission.
+Portions of this document, including but not limited to the Ripple logo,
+images and image templates are the property of Ripple Labs and cannot be
```

Comment:
Happy to make the changes you mention but it's my understanding that the company is, officially, Ripple Labs and "Ripple" is a DBA. Should we get an official opinion on this?